examples/stream-type.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width" />
+    <title>Media Chrome Stream Type Example</title>
+    <script type="module" src="../dist/index.js"></script>
+    <style>
+      /** add styles to prevent CLS (Cumulative Layout Shift) */
+      media-controller:not([audio]) {
+        display: block;         /* expands the container if preload=none */
+        max-width: 720px;       /* allows the container to shrink if small */
+        aspect-ratio: 16 / 9;   /* set container aspect ratio if preload=none */
+      }
+
+      video {
+        width: 100%;      /* prevents video to expand beyond its container */
+      }
+
+      .examples {
+        margin-top: 20px;
+      }
+
+      .stream-type {
+        display: none;
+        font-size: 24px;
+        line-height: 30px;
+        font-weight: bold;
+        color: #fff; 
+      }
+
+      media-controller[media-stream-type=live] .live {
+        display: block;
+      }
+
+      media-controller[media-stream-type=on-demand] .on-demand {
+        display: block;
+      }
+
+    </style>
+  </head>
+  <body>
+    <main>
+
+      <media-controller default-stream-type>
```

Comment:
suggestion(non-blocking): Since this isn't linked on the main `index.html` for the examples, I think this is fine, but we can/should provide some "real boy" examples of live and on demand (maybe after we do a fast follow/spike on supporting this in mux-video+mux-audio.

Reply:
Oof, forgot to clean this one up. I was gonna just drop the example html file and rely on the docs.